pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,21 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        # GH-25905
+        data_df = pd.DataFrame([
+            {'a': '1',   'b': '16.5%', 'c': 'test'},
+            {'a': '2.2', 'b': '15.3',  'c': 'another_test'}])
+        expected = pd.DataFrame([
+            {'a': '1',   'b': '16.5%', 'c': 'test'},
+            {'a': '2.2', 'b': '15.3',  'c': 'another_test'}],
+            dtype='float64')
+        type_dict = {'a': 'float64', 'b': 'float64', 'c': 'object'}
+
+        result = data_df.astype(dtype=type_dict, errors='ignore')
+
+        tm.assert_frame_equal(expected, result)
```

Comment:
Just swap result and expected here to be more conventional / readable

Reply:
Done in 343e26cb5578665299fdf1fbccae934f3c94bdd8.